packages/react-aria-components/docs/Tree.mdx

```diff
@@ -237,19 +237,530 @@ let items = [
+      {/*- begin highlight -*/}
+      <TreeItem id="venisaur" textValue="Venisaur" onAction={() => alert(`Opening Venisaur...`)}>
+      {/*- end highlight -*/}
+        <MyTreeItemContent>
+          Venisaur
+        </MyTreeItemContent>
+      </TreeItem>
+    </TreeItem>
+  </TreeItem>
+</Tree>
+```
+
+
+### Links
+
+Tree items may also be links to another page or website. This can be achieved by passing the `href` prop to the `<TreeItem>` component. Links behave the same way as described above for item actions depending on the `selectionMode` and `selectionBehavior`.
+
+```tsx example
+<Tree aria-label="Tree with onAction applied on the items directly" style={{height: '200px'}} defaultExpandedKeys={['bulbasaur', 'ivysaur']}>
+  <TreeItem id="bulbasaur" textValue="Bulbasaur" href="https://pokemondb.net/pokedex/bulbasaur">
+    <MyTreeItemContent>
+      Bulbasaur
+    </MyTreeItemContent>
+    <TreeItem id="ivysaur" textValue="Ivysaur" href="https://pokemondb.net/pokedex/ivysaur">
+      <MyTreeItemContent>
+        Ivysaur
+      </MyTreeItemContent>
+      <TreeItem id="venisaur" textValue="Venisaur" href="https://pokemondb.net/pokedex/venusaur">
+        <MyTreeItemContent>
+          Venisaur
+        </MyTreeItemContent>
+      </TreeItem>
+    </TreeItem>
+  </TreeItem>
+</Tree>
+```
+
+```css hidden
+.react-aria-TreeItem[data-href] {
+  cursor: pointer;
+}
+```
+
+#### Client side routing
+
+The `<TreeItem>` component works with frameworks and client side routers like [Next.js](https://nextjs.org/) and [React Router](https://reactrouter.com/en/main). As with other React Aria components that support links, this works via the <TypeLink links={docs.links} type={docs.exports.RouterProvider} /> component at the root of your app. See the [client side routing guide](routing.html) to learn how to set this up.
+
+## Disabled items
+
+A `TreeItem` can be disabled with the `isDisabled` prop. This will disable all interactions on the item,
+unless the `disabledBehavior` prop on `Tree` is used to change this behavior.
+Note that you are responsible for the styling of disabled items, however, the selection checkbox will be automatically disabled.
+
+```tsx example
+<Tree aria-label="Tree with disabled items" style={{height: '100px'}} defaultExpandedKeys={['bulbasaur']}>
+  <TreeItem id="bulbasaur" textValue="Bulbasaur">
+    <MyTreeItemContent>
+      Bulbasaur
+    </MyTreeItemContent>
+    {/*- begin highlight -*/}
+    <TreeItem id="ivysaur" textValue="Ivysaur" isDisabled>
+    {/*- end highlight -*/}
+      <MyTreeItemContent>
+        Ivysaur
+      </MyTreeItemContent>
+      <TreeItem id="venisaur" textValue="Venisaur">
+        <MyTreeItemContent>
+          Venisaur
+        </MyTreeItemContent>
+      </TreeItem>
+    </TreeItem>
+  </TreeItem>
+</Tree>
+```
+
+By default, only item selection is disabled. When `disabledBehavior` is set to `all`, all interactions such as focus, dragging, and actions are also disabled.
```

Comment:
This does not seem true. The entire item is disabled by default (the default is different from Table so maybe this was copied from there). The below example should also be updated to set `disabledBehavior="selection"` like GridList.